### PR TITLE
chore(operator, yaml): add openebs operator 2.1.0

### PR DESCRIPTION
**Pull request by akhilerm**
- add openebs operator 2.1.0
- add cstor operator 2.1.0 and dependents
- add zfs localPV 1.0.0 operator
- add openebs operator lite
- add openebs operator for ARM
- add openebs operator for suse caas platform

Signed-off-by: Akhil Mohan <akhil.mohan@mayadata.io>

### CODE REVIEW

**Review by pawanpraka1 — APPROVED** [review]
operator yaml looks good for ZFS-LocalPV.

**Review by prateekpandey14 — COMMENTED** [review]
lgtm

**Review by prateekpandey14 — APPROVED** [review]
lgtm

**Comment by akhilerm** [comment]
Added operator yamls for 2.1.0-RC2

**Review by pawanpraka1 — APPROVED** [review]
zfs operator yaml looks good.

**Comment by akhilerm** [comment]
Added operator yamls for 2.1.0

File: 2.1.0/cstor-operator-2.1.0.yaml [hunk 1]
```diff
@@ -0,0 +1,3935 @@
...
+            - name: OPENEBS_CSI_API_URL
+              value: https://openebs.io
+              # OpenEBS namespace where the openebs cstor operator components
+              # has been installed
+            - name: OPENEBS_NAMESPACE
+              value: openebs
+              # Enable/Disable auto-remount feature, when volumes
+              # recovers form the read-only state
+            - name: REMOUNT
+              value: "false"
```
Comment (mittachaitu) [inline 2.1.0/cstor-operator-2.1.0.yaml:1089]: @payes Do we need to enable it by default?

**Pull request merged**